src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -86,7 +86,7 @@ const ProfilePicture = ({
                   style={styles.profileImage(size, width)}
                   source={{
                     uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
-                    cache: 'cacheOnly'
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The same comment applies here as well. Changing the cache policy from `'cacheOnly'` to `'web'` may increase the load time of the profile picture. Consider using a different caching strategy like `'immutable'`.
```diff
- cache: 'web'
+ cache: 'immutable'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->